### PR TITLE
[fix] add `device` argument to the call of `modeling_opt._make_causal_mask`

### PR DESCRIPTION
**Pull request by shionhonda**
This PR fixes a bug in the call of `modeling_opt._make_causal_mask()` in `OPTModelBranch`. This private method requires `device` argument, as found in:
https://github.com/huggingface/transformers/blob/b29fd6971d9cd6ba2a824628effe243f543b8f61/src/transformers/models/opt/modeling_opt.py#L68-L70

### CODE REVIEW

**Review by jon-tow — APPROVED** [review]
Thanks for the quick fix, @shionhonda!

**Comment by jon-tow** [comment]
This update is only in the latest `main` branch, so let's wait until a patch release is available to merge this. For now, try to use an earlier version of `transformers`, if possible 🙏

**Comment by shionhonda** [comment]
Ah, I see! I was looking at the latest main branch. This change in the API happened very recently.

**Pull request closed without merging**
